File: bank(csa2)/bankServer/src/main/java/com/jbosak/csa/bank/server/Session.java
package com.jbosak.csa.bank.server;

import java.sql.Timestamp;
import java.util.Date;

public class Session {

    private Timestamp expirationTime;
    private User user;

    public Session(User user) {
        this.user = user;
        expirationTime = new Timestamp((new Date().getTime() + 1000000));
    }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }

    public Timestamp getExpirationTime() {
        return expirationTime;
    }

    public void setExpirationTime(Timestamp expirationTime) {
        this.expirationTime = expirationTime;
    }
}

File: bank(csa2)/bankServer/src/main/kotlin/com/jbosak/csa/bank/server/Session.kt
package com.jbosak.csa.bank.server

import java.io.Serializable
import java.sql.Timestamp
import java.util.*

class Session(var user: User?): Serializable {

    var expirationTime: Timestamp? = null

    init {
        expirationTime = Timestamp(Date().time + 1000000)
    }
}
